package.json

```diff
@@ -36,31 +47,51 @@
   "author": "MuleSoft, Inc.",
   "license": "Apache-2.0",
   "bugs": {
-    "url": "https://github.com/mulesoft-labs/raml-javascript-generator/issues"
+    "url": "https://github.com/mulesoft-labs/raml-js-client-generator/issues"
   },
-  "homepage": "https://github.com/mulesoft-labs/raml-javascript-generator",
+  "homepage": "https://github.com/mulesoft-labs/raml-js-client-generator",
   "devDependencies": {
-    "blue-tape": "^0.2.0",
+    "@commitlint/cli": "^3.1.2",
+    "@commitlint/config-angular": "^3.1.1",
+    "@commitlint/prompt": "^3.1.2",
     "browserify": "^13.0.0",
+    "browserify-istanbul": "^0.2.1",
+    "chai": "^4.1.1",
+    "commitizen": "^2.9.6",
     "envify": "^3.4.0",
+    "eslint": "^4.5.0",
+    "eslint-config-airbnb-base": "^11.3.2",
+    "eslint-plugin-import": "^2.7.0",
     "express": "^4.12.3",
+    "husky": "^0.14.3",
     "istanbul": "^1.0.0-alpha.2",
+    "karma": "^1.7.0",
+    "karma-browserify": "^5.1.1",
+    "karma-coverage": "^1.1.1",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-mocha": "^1.3.0",
     "methods": "^1.1.2",
+    "mocha": "^3.5.0",
     "nock": "^7.5.0",
     "sinon": "^1.16.1",
-    "tap-spec": "^4.1.1",
-    "tape-run": "^2.1.0",
-    "typescript": "^1.8.9",
-    "typings": "^0.7.9"
+    "tslint": "^5.7.0",
+    "tslint-config-airbnb": "^5.2.1",
+    "typescript": "^2.4.2",
+    "validate-commit-msg": "^2.14.0",
+    "watchify": "^3.9.0"
   },
   "dependencies": {
```

Comment:
we should enable greenkeeper.io if it's not enabled already

Reply:
is greenkeeper.io enabled in any project? not sure I have enough credentials to enable that on this project